views/overlays/admin.hbs

```diff
@@ -122,6 +142,8 @@
 		document.getElementById('commentators').value = data.commentators || "";
 		document.getElementById('lscore').value = data.lscore || 0;
 		document.getElementById('rscore').value = data.rscore || 0;
+		window.characterLeft = data.lCharacter || null;
+		window.characterRight = data.rCharacter || null;
```

Comment:
Could these be `var`s defined at the top of the `<script>` instead? It'll make it easier to pull all this stuff out into separate JS files in the future, and avoid a global variable.